lib/widget/camera.dart

```diff
@@ -236,7 +306,7 @@ class _CameraScreenState extends State<CameraScreen> {
           textbutton(
             title: 'Pick from gallery',
             onPressed: () {
```

Comment:
Never show error when you don't have to. If useGallery is false, simply don't display the button.

Reply:
i fixed according to your suggestion